modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormAdminDisplayContext.java

```diff
@@ -164,7 +166,9 @@ public DDMFormAdminDisplayContext(
 		DDMStorageAdapterTracker ddmStorageAdapterTracker,
 		DDMStructureLocalService ddmStructureLocalService,
 		DDMStructureService ddmStructureService, JSONFactory jsonFactory,
-		NPMResolver npmResolver, Portal portal) {
+		NPMResolver npmResolver,
+		ObjectDefinitionLocalService objectDefinitionLocalService,
+		Portal portal) {
```

Comment:
@mlan0 please fix the `DDMFormAdminDisplayContextTest.java` [here](https://github.com/carolmariaabb/liferay-portal/blob/984b54552641844507fd25476a9a81b96fc48a0f/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/test/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormAdminDisplayContextTest.java#L285-L309). You can pass `null` instead of mocking a `ObjectDefinitionLocalService`.